examples/tutorials/research/01_introduction_to_research.ipynb

```diff
@@ -0,0 +1,2478 @@
+{
```

Comment:
<p>Remove all unnecessary files and directories from the output.</p><p>You have run <code>vgg7_research</code>. So that's the only thing that should appear in the output.</p><p>Otherwise it remains unclear what's going on and why.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/337/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>